internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -4464,6 +4464,41 @@ func TestHandleGetInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleGetMempoolInfo(t *testing.T) {
+	t.Parallel()
+	txDescOne := &mempool.TxDesc{
+		StartingPriority: 0,
+	}
+	txDescOne.Tx = dcrutil.NewTx(block432100.Transactions[0])
+	txDescOne.Type = stake.TxTypeSSGen
+	txDescOne.Added = time.Time{}
+	fee, _ := dcrutil.NewAmount(0.00000001)
```

Comment:
Not needed to parse that.  You could just do `fee := dcrutil.Amount(1)` or skip it altogether as I showed above since it's just being converted to an `int64` anyway.